musicbrainz/getArtists.js

```diff
@@ -71,23 +83,46 @@ module.exports = {
                             placeOfDeath = aboutArtist.end_area.name;
                         }
 
+                        var pseudonym = [];
+                        if (aboutArtist.hasOwnProperty("aliases") && aboutArtist.aliases != null) {
+                            aboutArtist.aliases.forEach(function (alias) {
+                                pseudonym.push(alias.name);
+                            });
+                        }
+
+                        var tags = [];
+                        var artist_type = null;
+                        if (aboutArtist.hasOwnProperty("tags") && aboutArtist.tags != null) {
+                            aboutArtist.tags.forEach(function (tag) {
+                                if (tag.name != "to clean up") {
+                                    tags.push(tag.name);
+                                    if (tag.name == "composer") {
+                                        artist_type = "composer";
+                                    }
+                                }
+
+                            });
+                        }
+                        var work, release, instrument = [];
+
+
                         artistsAPI.push({
                             name: name,
+                            artist_type: artist_type,
                             nationality: nationality,
                             dateOfBirth: dateOfBirth,
                             dateOfDeath: dateOfDeath,
                             placeOfBirth: placeOfBirth,
                             placeOfDeath: placeOfDeath,
                             instrument: null,
-                            psuedonym: null,
+                            psuedonym: pseudonym,
```

Comment:
@ShilpaGhanashyamGore Yep you're right, already changed that but forgot to commit it!
@LukasNavickas We do! If you check the scraped json, there is an element called "aliases"